clients/fides-js/src/lib/initialize.ts

```diff
@@ -408,14 +410,37 @@ export const initialize = async ({
   const { fides_meta, identity, fides_string, tcf_consent } = fides.cookie;
 
   // used to set Fides.consent
-  const consent = applyOverridesToConsent(
+  const consentObject = applyOverridesToConsent(
     fides.cookie.consent,
     fides.experience?.non_applicable_privacy_notices,
     fides.experience?.privacy_notices,
     options.fidesConsentFlagType ?? undefined,
     options.fidesConsentNonApplicableFlagMode ?? undefined,
   );
 
+  const hasPrivacyNotices =
+    !!fides.experience?.non_applicable_privacy_notices ||
+    !!fides.experience?.privacy_notices;
+
+  // Any unknown consent values should return falsy values automatically
```

Comment:
Checking my understanding:

1. When an unknown consent flag is accessed we return `false`.
2. If an experience does not load that means all consent flags are "unknown".
3. Therefore, when experiences fail to load, this code is in play "by default" to say that the user has not given consent.